app/views/cfp/welcome/not_existing.html.haml

```diff
@@ -1,28 +1,26 @@
 %section
   .page-header
-    %h1 Welcome
+    %h1= t('cfp.welcome')
   .row
     .span16
   .row
     .span16
       .flash
         %div{class: "alert-message error fade in", "data-alert" => "alert"}
-          %p
-            Thank you for your interest in participating in #{@conference.title}.
-          %p
-            Unfortunately, the Call for Participation does not (yet?) exist.
+          %p= t('cfp.pages.thank_for_interest', {conference: @conference.title})
+          %p= t('cfp.pages.inexistent_cfp')
```

Comment:
isn't it "unexisting"?

Reply:
i'm looking at it right now, thanks !

Reply:
From the [collins dictionnary](https://www.collinsdictionary.com/dictionary/english/inexistent), _inexistent_ seems ok in en_US, but seems to be _a rare word for "nonexistent"_ in en_UK, so i'll go with `nonexistant` then.